userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -1254,25 +1449,30 @@ image::ProcessChargeback.png[width=450,align="center"]
 . Fill in the *Comment* field (optional). The text you enter here displays on the <<Timeline>> page.
 . Click the *Save* button. Kaui saves the chargeback and displays it as a separate row on the Payments Details page:
 
-image::PaymentDetails-Chargeback-Labeled.png[width=650,align="center"]
+image::PaymentDetails-Chargeback-Labeled.png[width=850,align="center"]
 
 === Timeline
 
-*_IN PROCESS_*
+The Timeline displays a chronological list of events (that is, an audit log) that occurred for a specific customer account. You cannot make changes to the timeline.
+
+To see the Timeline, open an account and click *Timeline* on the sub-menu:
+
+image::TimelinePage.png[width=850,align="center"]
 
-// How to view/interpret the timeline. What it shows.
+On the Timeline page, you can:
 
-// The Timeline also includes information about payment retries (When an account is unpaid, Kill Bill (the payment retry system) retries payments per the scheduled configuration.
+* _Filter the events_: Make selections in the *Filter by event* or *Filter by bundle* fields to display only specific events (e.g., `PURCHASE`, `INVOICE`) and/or by a specific subscription bundle.
 

```

Comment:
Minor correction: `Filter by event` should be `Filter by event type`

Reply:
This has been fixed